I1204 11:59:14.037946 4607114752 bigquery_client.py:1144] Requesting discovery document from https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest
I1204 11:59:14.209074 4607114752 model.py:111] --request-start--
I1204 11:59:14.209172 4607114752 model.py:112] -headers-start-
I1204 11:59:14.209224 4607114752 model.py:114] accept: */*
I1204 11:59:14.209270 4607114752 model.py:114] accept-encoding: gzip, deflate
I1204 11:59:14.209311 4607114752 model.py:114] user-agent: google-api-python-client/1.7.10 (gzip)
I1204 11:59:14.209352 4607114752 model.py:115] -headers-end-
I1204 11:59:14.209392 4607114752 model.py:116] -path-parameters-start-
I1204 11:59:14.209433 4607114752 model.py:118] tableId: weatherDataTable
I1204 11:59:14.209474 4607114752 model.py:118] projectId: test456depp
I1204 11:59:14.209512 4607114752 model.py:118] datasetId: weatherData
I1204 11:59:14.209553 4607114752 model.py:119] -path-parameters-end-
I1204 11:59:14.209592 4607114752 model.py:120] body: None
I1204 11:59:14.209671 4607114752 model.py:121] query: ?
I1204 11:59:14.209713 4607114752 model.py:122] --request-end--
I1204 11:59:14.209824 4607114752 discovery.py:868] URL being requested: DELETE https://bigquery.googleapis.com/bigquery/v2/projects/test456depp/datasets/weatherData/tables/weatherDataTable?
I1204 11:59:14.435619 4607114752 model.py:182] --response-start--
I1204 11:59:14.435803 4607114752 model.py:184] vary: Origin, X-Origin, Referer
I1204 11:59:14.435904 4607114752 model.py:184] content-type: application/json; charset=UTF-8
I1204 11:59:14.435990 4607114752 model.py:184] date: Fri, 04 Dec 2020 16:59:14 GMT
I1204 11:59:14.436069 4607114752 model.py:184] server: ESF
I1204 11:59:14.436146 4607114752 model.py:184] cache-control: private
I1204 11:59:14.436221 4607114752 model.py:184] x-xss-protection: 0
I1204 11:59:14.436296 4607114752 model.py:184] x-frame-options: SAMEORIGIN
I1204 11:59:14.436372 4607114752 model.py:184] x-content-type-options: nosniff
I1204 11:59:14.436448 4607114752 model.py:184] alt-svc: h3-29=":443"; ma=2592000,h3-T051=":443"; ma=2592000,h3-Q050=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000,quic=":443"; ma=2592000; v="46,43"
I1204 11:59:14.436522 4607114752 model.py:184] transfer-encoding: chunked
I1204 11:59:14.436598 4607114752 model.py:184] status: 404
I1204 11:59:14.436671 4607114752 model.py:184] content-length: 286
I1204 11:59:14.436745 4607114752 model.py:184] -content-encoding: gzip
I1204 11:59:14.436820 4607114752 model.py:186] b'{\n  "error": {\n    "code": 404,\n    "message": "Not found: Dataset test456depp:weatherData",\n    "errors": [\n      {\n        "message": "Not found: Dataset test456depp:weatherData",\n        "domain": "global",\n        "reason": "notFound"\n      }\n    ],\n    "status": "NOT_FOUND"\n  }\n}\n'
I1204 11:59:14.436900 4607114752 model.py:187] --response-end--
